# joy timeout in seconds
joy_timeout: 0.5

# define trigger positions in axes array of gamepad (used if a mapping of trigger axes values from [-1,1] to [0,1] i requested) -->
left_trigger: 4
right_trigger: 5

